Minor editorial, grammatical, and spelling improvments to docs (#1701)
vercel/turborepo: docs/pages/docs/guides/migrate-from-lerna.mdx

![Turborepo vs. Lerna Task Execution](/images/docs/turbo-vs-lerna-execution.png)

> The above may not not look like much. But imagine that A's `build` task takes 20 minutes and now look at when B's tests run. Whoa....yeah....scheduling can save huge amounts of time when you have long build times.

Reviewer: ```suggestion
The above may not look like much. But imagine that A's `build` task takes 20 minutes, and now look at when B's tests run. Scheduling can save huge amounts of time when you have long build times.
```

### Package Publishing, Versioning, and Changelog Generation

> Lerna can version and publish packages to npm registries as well as create changelogs. It's extremely good at this, especially for releasing canary pre-releases. While Turborepo may eventually tackle these features, it is not a high priority goal &mdash; we suggest using [Changesets](https://github.com/changesets/changesets) in the meantime. If Changesets isn't for you, and you want to stick with Lerna's publish flow, you can use Lerna and Turborepo together.

Reviewer: ```suggestion
Lerna can version and publish packages to npm registries and create changelogs. It is extremely good at this, especially for releasing canary pre-releases. While Turborepo may eventually tackle these features, it is not a high-priority goal, and we suggest using [Changesets](https://github.com/changesets/changesets) in the meantime. If Changesets isn't for you, and you want to stick with Lerna's publish flow, you can use Lerna and Turborepo together.
```

}
```

> If you aren't using yarn or npm workspaces with Lerna, you should keep around your `lerna bootstrap` command if you have one. Turborepo considers Lerna a valid workspace implementation. Everything should work as expected.

Reviewer: ```suggestion
If you are not using yarn or npm workspaces with Lerna, you should keep around your `lerna bootstrap` command if you have one. Turborepo considers Lerna a valid workspace implementation. Everything should work as expected.
```
